Cargo.toml

```diff
@@ -14,11 +14,13 @@ geo-types = { version = "0.7.4", optional = true }
 num-traits = "0.2"
 serde = { version = "1.0", default-features = false, optional = true }
 thiserror = "1.0.23"
+log = "0.4.17"
 
 [dev-dependencies]
 criterion = { version = "0.2" }
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_json = "1.0"
+env_logger = "0.9.0"
```

Comment:
Is this used?

Reply:
I only used it locally. I'll remove it.

Reply:
I think this is a stale diff - it was removed in https://github.com/georust/wkt/pull/103/commits/5cdefc65794d1dad9022ab79984ecdcc2c60dc4d